<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yeastar-PC" name="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup" tests="26" failures="2" timestamp="20 Apr 2018 22:46:22 GMT" time="1379.564" errors="0">
  <testcase name="F8_check_Voicemail" time="35.195" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="19.132" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F1_FailtoHangup" time="90.453" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F6_FailtoConference" time="52.390" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup">
    <failure type="java.lang.AssertionError" message="CDR被叫方检测 【预期：1000 [6207(6400)] 实际：6400 [6207(6400)]】">
      <![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1000 [6207(6400)] 实际：6400 [6207(6400)]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:263)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:252)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup.F6_FailtoConference(RingGroup.java:709)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F6_FailtoConference -->
  <testcase name="A2_AddRingGroups" time="21.292" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="20.483" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="15.131" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F2_FailtoExtension" time="61.074" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="B_EditRingGroupFxs" time="25.612" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="H2_recovery" time="34.486" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="E2_RingAll" time="66.948" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="19.948" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="E1_Sequentially" time="32.491" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="H1_recovery" time="31.865" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="G_delete" time="77.307" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="D_InboundtoRingGroup_a" time="370.867" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F4_FailtoRingGroup" time="74.391" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A1_Init" time="22.980" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F7_FailtoVoicemail" time="59.863" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="18.374" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="C_EditInRoute1" time="34.741" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F3_FailtoIVR" time="54.419" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A3_add_cancel" time="20.705" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="A2_AddRingGroups" time="21.118" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
  <testcase name="F5_FailtoQueue" time="78.048" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup">
    <failure type="java.lang.AssertionError" message="CDR被叫方检测 【预期：1100 [6700(1100)] 实际：6700 [6700(1100)]】">
      <![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1100 [6700(1100)] 实际：6700 [6700(1100)]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:263)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:252)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup.F5_FailtoQueue(RingGroup.java:689)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- F5_FailtoQueue -->
  <testcase name="A2_AddRingGroups" time="20.251" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.DetailCase.pbxcase.RingGroup -->
